javacutil/src/main/java/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -384,8 +384,7 @@ public static boolean isBoxedPrimitive(TypeMirror type) {
     public static boolean isImmutableTypeInJdk(TypeMirror type) {
         return isPrimitive(type)
                 || (type.getKind() == TypeKind.DECLARED
-                        && ImmutableTypes.isImmutable(
-                                getQualifiedName((DeclaredType) type).toString()));
+                        && ImmutableTypes.isImmutable(getQualifiedName((DeclaredType) type)));
```

Comment:
There are a few places where `toString` is called on the result of `getQualifiedName`.
I think earlier `getQualifiedName` returned a `Name` and not a `String`.
I'm wondering whether we should either go back to using `Name` or remove the redundant `toString` calls throughout.
What do you think?

Reply:
I think we don't benefit from the extra properties in `Name`, so it's just more intuitive to use `String` instead. I have removed the redundant `toString` calls.